src/xai_grok/schemas/function.py

```diff
@@ -0,0 +1,6 @@
+import pydantic
+
+
+class Function(pydantic.BaseModel):
+    name: str
+    arguments: str
```

Comment:
The `arguments` field is defined as a string, but should likely be a more structured type (like a dictionary or list) to properly represent function arguments with their types and values.